hat/templates/iaso/base.html

```diff
@@ -26,6 +26,11 @@
   <meta property="og:image" content="" />
   <meta property="og:url" content="{{ request.build_absolute_uri }} " />
   {% endif %}
+  {% if ANALYTICS_SCRIPTS %}
+    {% for analytic_script in ANALYTICS_SCRIPTS %}
```

Comment:
You can't have all the analytics script of all accounts displayed here, that defeats the point of having a different script per account

Reply:
Ah this seems to be about the same thing I'm commenting below

Reply:
As you can see the analytics script will be added only for the embedded calendar, which is accessible by not authenticated user. And on the embedded calendar there can be displayed campaigns from different account, that's why I tried to get all the analytics script from all accounts. I don't know if it's clear or if you want we can have a 5min call. thanks